packages/server/src/note/parser.ts

```diff
@@ -1,16 +1,16 @@
 import {Field, Note} from '../../../shared/type';
-import {loadFront} from 'yaml-front-matter';
+import matter from 'gray-matter';
 import {Either, isLeft, left, map, right} from 'fp-ts/lib/Either';
 import {expectNever, requiredFFVersionRegex} from '../util/util';
 import semver from 'semver';
 import {pipe} from 'fp-ts/lib/pipeable';
 
 export const parseNote = (fields: Field[], note: string): Either<ParseError, Note> => {
-    const {__content: content, ...meta} = loadFront(note);
+    const {content: content, ...meta} = matter(note);
```

Comment:
```suggestion
    const {content, data} = matter(note);
```
Dann muss man weiter unter nicht immer meta.data schreiben, sondern nur data